CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.2.2
```

Comment:
This should be 'Unreleased' until a new version is released (which might be 2.2.2, but could equally be 2.3) – it also makes it clear to users looking at the changelog that these changes haven't gone out yet.